src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -1309,6 +1329,70 @@
 short-summary: Retrieves latest restorable timestamp for the given sql container in given region.
 """
 
+# gremlin restorable resources
+helps['cosmosdb gremlin restorable-database'] = """
+type: group
+short-summary: Manage different versions of gremlin databases that are restorable in a Azure Cosmos DB account.
+"""
+
+helps['cosmosdb gremlin restorable-database list'] = """
+type: command
+short-summary: List all the versions of all the gremlin databases that were created / modified / deleted in the given restorable account.
+"""
+
+helps['cosmosdb gremlin restorable-graph'] = """
+type: group
+short-summary: Manage different versions of gremlin graphs that are restorable in a database of a Azure Cosmos DB account.
+"""
+
+helps['cosmosdb gremlin restorable-graph list'] = """
+type: command
+short-summary: List all the versions of all the gremlin graphs that were created / modified / deleted in the given database and restorable account.
+"""
+
+helps['cosmosdb gremlin restorable-resource'] = """
+type: group
+short-summary: Manage the databases and its graphs that can be restored in the given account at the given timestamp and region.
+"""
+
+helps['cosmosdb gremlin restorable-resource list'] = """
+type: command
+short-summary: List all the databases and its graphs that can be restored in the given account at the given timestamp and region.
+"""
+
+# table restorable resources
+helps['cosmosdb table restorable-table'] = """
+type: group
+short-summary: Manage different versions of tables that are restorable in Azure Cosmos DB account.
```

Comment:
`cosmosdb table list-restorable-table`

Reply:
Hi Yishi, we are doing this way so that we are compatible with existing Api types like SQL and MongoDB.